src/main/java/com/peerrequest/app/model/Category.java

```diff
@@ -0,0 +1,96 @@
+package com.peerrequest.app.model;
+
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.ZonedDateTime;
+
+
+/**
+ * This class represents a category.
+ *
+ * @author Ludger Halpick
+ * @author Michael Mildt
+ * @version 0.0.1
+ */
+public class Category {
```

Comment:
relation to Entry.java missing

Reply:
Is not needed . All related entries are identified by their category attribute